### DISCUSSION TITLE
Version 1.25.0

### DISCUSSION THREAD

**Discussion by MM2-0** [MM2-0/Kvaesitso#394] [Announcements]
- Added note widget
  - a widget that holds a simple text note
  - supports basic markdown
  - notes can be exported as markdown file
- The card background can now be hidden for app widgets
- Added a preference to hide the "hidden items" button from the search bar
- Bugfixes and optimizations

<hr /><em>This discussion was created from the release <a href='https://github.com/MM2-0/Kvaesitso/releases/tag/v1.25.0'>Version 1.25.0</a>.</em>

**Comment by rahaaatul** [comment]
An integration with Obsidian and it will be 🔥

**Comment by rahaaatul** [comment]
Btw, Can you hide symbols when the cursor is not on the modified text?

https://github.com/MM2-0/Kvaesitso/assets/86149861/27306625-57c2-4087-91ba-313cea0edfa5

**Reply by MM2-0** [reply]
Press back / perform a back gesture to unfocus the widget

**Reply by rahaaatul** [reply]
Hey Owen, just wanted to say I'm loving this new feature! It's super handy for quick notes. No more waiting for Keep Notes to load, yay! It's definitely making things faster and easier.

By the way, I have a cool idea! How about adding an auto-save feature to sync in Obsidian Vault? It would be awesome to edit notes in Obsidian later without any hassle. Oh, and it would be really neat if we could have an "Edit with Obsidian" option right in the note widget's hamburger menu. Keep up the great work!

Thank YOU 🙏

**Reply by FeralFlora** [reply]
I think Obsidian (or any markdown editor) integration could be achieved by have a setting to specify a default note folder. This can then be set to a folder inside an Obsidian vault. Then, like @rahaaatul mentioned, notes would save to file there automatically.

**Reply by jspivey40** [reply]
> I think Obsidian (or any markdown editor) integration could be achieved by have a setting to specify a default note folder. This can then be set to a folder inside an Obsidian vault. Then, like @rahaaatul mentioned, notes would save to file there automatically.

This is a very good idea, and likely would also make it compatible with other apps, similar to Obsidian. Excellent idea!

**Reply by MM2-0** [reply]
I have added a file sync option (available in nightlies tomorrow) which allows you to sync a note widget instance with a file. I plan to add an option to sync an entire folder later.
